src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.figures;
+
+public class Figure implements Drawable {

```

Comment:
it should also implement the interface with method for counting the area of each figure